Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -915,10 +915,10 @@ def get_team_membership(team_id: Union[int, str], user_name: str) -> dict:
     return response
 
 
-def get_team_members(organization: str, team_slug: str) -> list:
+def get_team_members(organization: str, team_slug: str, limit: int = 10) -> list:
     page = 1
     results = []
-    while True:
+    while page <= limit:
```

Comment:
Since the purpose of the command is to get members, it makes more sense to limit the number of users and not the number of pages.
I think it would be better to use the limit for that.